configurations/batch/batch.ini.template

```diff
@@ -68,6 +68,7 @@ enabledWorkers.KAsyncCopyCaptions					= 1
 enabledWorkers.KAsyncCopyCuePoints                   = 1
 enabledWorkers.KSyncReachCreditTaskRunner	    = 0
 enabledWorkers.KAsyncExportCsv                      = 0
+enabledWorkers.KAsyncLiveEntryArchive               = 1
```

Comment:
Any reason this was added to the template? Do we really need to run this batch on all prod machines?

Reply:
We added it to work as LiveToVOD

Reply:
So why do you need another worker if this is part of the liveTovod process?
Am I missing something?

Reply:
It has two scenarios: 
1) Work automatically after the live to VOD finished
2) Work when a proper API call was triggered